Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM       ubuntu:13.10
+FROM       golang:latest
```

Comment:
I think we should keep the Go versions in sync between the Makefile and the Dockerfile.